src/Microsoft.OData.Core/Build.Net35/Microsoft.OData.Core.NetFX35.csproj

```diff
@@ -1545,6 +1539,24 @@
     <Compile Include="..\UnknownEntitySet.cs">
       <Link>Microsoft\OData\Core\UnknownEntitySet.cs</Link>
     </Compile>
+    <Compile Include="..\UriParser\Aggregation\AggregateExpressionToken.cs">

```

Comment:
I replaced with ODataCrossTargettingSourcePath, as the other references in this file.